DQM/BeamMonitor/plugins/OnlineBeamMonitor.cc

```diff
@@ -255,19 +272,25 @@ std::shared_ptr<onlinebeammonitor::NoCache> OnlineBeamMonitor::globalBeginLumino
         if (beamSpotsMap_.find("HLT") != beamSpotsMap_.end() &&
             beamSpotsMap_["Transient"].x0() == beamSpotsMap_["HLT"].x0()) {
           lastLumiAnalyzed_ = lastLumiHLT_;
+          startTimeStamp_ = startTimeStampHLT_;
+          stopTimeStamp_ = stopTimeStampHLT_;
+          lumiRange_ = lumiRangeHLT_;
 
         } else if (beamSpotsMap_.find("Legacy") != beamSpotsMap_.end() &&
                    beamSpotsMap_["Transient"].x0() == beamSpotsMap_["Legacy"].x0()) {
           lastLumiAnalyzed_ = lastLumiLegacy_;
+          startTimeStamp_ = startTimeStampLegacy_;
+          stopTimeStamp_ = stopTimeStampLegacy_;
+          lumiRange_ = lumiRangeLegacy_;
         }
       }
 
       outFile.open(tmpname.c_str());
 
       outFile << "Runnumber " << frun << " bx " << 0 << std::endl;
-      //outFile << "BeginTimeOfFit " << fbeginTimeOfFit << " " << 0 << std::endl;
-      //outFile << "EndTimeOfFit " << fendTimeOfFit << " " << 0 << std::endl;
-      outFile << "LumiRange " << lastLumiAnalyzed_ << " - " << lastLumiAnalyzed_ << std::endl;
+      outFile << "BeginTimeOfFit " << startTimeStamp_ << " " << 0 << std::endl;
+      outFile << "EndTimeOfFit " << stopTimeStamp_ << " " << 0 << std::endl;
+      outFile << "LumiRange " << lumiRange_ << " - " << lastLumiAnalyzed_ << std::endl;
```

Comment:
Given the fix I made in #37875, I think this should be changed to:
```suggestion
      outFile << "LumiRange " << lumiRange_ << std::endl;
```